docs/protocol/cli.md

```diff
@@ -522,6 +522,30 @@ $ screen -S farming -X quit
 
 If you are facing issues with your node/farmer you can try a few of the following things below, if you are unable to get your issue resolved please check our [Forums](https://forums.subspace.network) to see if your issue may have been solved, if its a new one feel free to post it! You can also join our [Discord](https://discord.gg/subspace-network) for additional Peer to Peer help.
 
+:::info
+
+We have included a few tutorials below that may help you in your support journey, this is not an all inclusive list but we welcome contributions
+
+:::
+
+### Wipe Node & Farmer
+
+:::tip Updated from a previous version and now having issues?
+
+Occasionally after updating to a new version of the CLI you will need to wipe your node and farmer, generally this should not be required but can be attempted if your farmer is having issues after having had worked fine previously.
+
+:::
+
+To simply restart the node, go to the terminal where you started the `farm` command, and press `Ctrl + C` you should see a shutdown message appear and the application will attempt a simple shutdown, if you dont see the message press `Ctrl + C` again to force shutdown. You can then simply start the farmer again with the `farm` command you used prior.
+
+Use the same file name as the previous init and farm steps, then add the wipe command to free the previous memory that was being used. Generally, only do this if you have severe errors and are prompted by a staff member.
+
+```bash
+./subspace-cli-file-name wipe
+```
+
+After wiping, follow the init and farm steps above to start farming again!
```

Comment:
The commands are backticked here, unleess im mistaken by what you mean?

```
` ` `bash
./subspace-cli-file-name wipe
` ` `
```

Above is what is currently present, (minus the spaces inbetween the backticks for the sake of escape issues in github comments)

In this instance we do the code blocks instead of inline backticks to provide the user with the copy button when rendered 

Codeblock backticks
![image](https://github.com/subspace/subspace-docs/assets/31865152/29a1d38d-cdad-4d6a-8411-d24cedaac2db)

Inline backticks
![image](https://github.com/subspace/subspace-docs/assets/31865152/8833c856-36c0-4029-8a51-efad33094c00)

Reply:
On `init` and `farm` as they are in the previous paragraph.

Reply:
Ahh, I see now 😅 thank you!

Reply:
Resolved with commit 5ca0e73